README.md

```diff
@@ -24,6 +25,9 @@ For example, the intent to launch the Practice Mode of the Tap app would have th
 
 `.PRACTICE` and `.HELP` don't have any arguments or expect any results.  Practice mode itself is relatively unstructured, and should let the patient practice as long as they want before exiting.  Help mode should display instructions or a guide to using the specific test, and then switch to Practice mode.
 
+`.HISTORY` has 1 argument:
+- user, which is a string of the current user's id
```

Comment:
It should say "Patient ID, which is a string of the current patient's id" to match the .TRIAL section